### PR TITLE
(WIP) Add manpages for more command-line utilities

### PR DESCRIPTION
**Pull request by sppmacd**
Already done (before this PR):
* [X] /usr/share/man/man1/adjtime.md
* [X] /usr/share/man/man1/avol.md
* [X] /usr/share/man/man1/base64.md
* [X] /usr/share/man/man1/beep.md
* [X] /usr/share/man/man1/cal.md
* [X] /usr/share/man/man1/crash.md
* [X] /usr/share/man/man1/date.md
* [X] /usr/share/man/man1/ddate.md
* [X] /usr/share/man/man1/echo.md
* [X] /usr/share/man/man1/expr.md
* [X] /usr/share/man/man1/find.md
* [X] /usr/share/man/man1/gml-format.md
* [X] /usr/share/man/man1/host.md
* [X] /usr/share/man/man1/jp.md
* [X] /usr/share/man/man1/js.md
* [X] /usr/share/man/man1/keymap.md
* [X] /usr/share/man/man1/ls.md
* [X] /usr/share/man/man1/man.md
* [X] /usr/share/man/man1/md.md
* [X] /usr/share/man/man1/mkdir.md
* [X] /usr/share/man/man1/mktemp.md
* [X] /usr/share/man/man1/open.md
* [X] /usr/share/man/man1/pape.md
* [X] /usr/share/man/man1/pmap.md
* [X] /usr/share/man/man1/printf.md
* [X] /usr/share/man/man1/readlink.md
* [X] /usr/share/man/man1/syscall.md
* [X] /usr/share/man/man1/tar.md
* [X] /usr/share/man/man1/test-js.md
* [X] /usr/share/man/man1/test.md
* [X] /usr/share/man/man1/uname.md
* [X] /usr/share/man/man1/unzip.md
* [X] /usr/share/man/man1/watch.md
* [X] /usr/share/man/man1/xargs.md
* [X] /usr/share/man/man1/zip.md
* [X] /usr/share/man/man8/chroot.md
* [X] /usr/share/man/man8/lspci.md
* [X] /usr/share/man/man8/modload.md
* [X] /usr/share/man/man8/modunload.md
* [X] /usr/share/man/man8/mount.md
* [X] /usr/share/man/man8/purge.md
* [X] /usr/share/man/man8/sysctl.md
* [X] /usr/share/man/man8/useradd.md
* [X] /usr/share/man/man8/userdel.md

Missing:
* [x] allocate
* [x] aplay
* [x] arp
* [x] basename
* [x] bt
* [x] cat
* [x] checksum
    * [ ] md5sum
    * [ ] sha1sum
    * [ ] sha256sum
    * [ ] sha512sum
* [x] chgrp
* [x] chmod
* [ ] chown
* [ ] chres
* [ ] clear
* [ ] copy
* [ ] cp
* [ ] CppParserTest
* [ ] cut
* [ ] dd
* [ ] df
* [ ] dirname
* [ ] disasm
* [ ] disk_benchmark
* [ ] dmesg
* [X] du
* [ ] env
* [ ] false
* [ ] fgrep
* [ ] flock
* [ ] fortune
* [ ] functrace
* [ ] grep
* [ ] gron
* [ ] gunzip
* [ ] gzip
* [ ] head
* [ ] hexdump
* [ ] hostname
* [ ] id
* [ ] ifconfig
* [ ] ini
* [ ] killall
* [ ] kill
* [ ] ln
* [ ] lsirq
* [ ] lsof
* [ ] misbehaving-application
* [ ] mkfifo
* [ ] mknod
* [X] more
* [X] mv
* [ ] nc
* [ ] nl
* [X] notify
* [ ] nproc
* [ ] ntpquery
* [ ] passwd
* [ ] paste
* [ ] pidof
* [ ] ping
* [ ] PreprocessorTest
* [ ] pro
* [ ] profile
* [ ] ps
* [ ] readelf
* [ ] realpath
* [ ] reboot
* [ ] rm
* [ ] rmdir
* [ ] seq
* [ ] shot
* [ ] shuf
* [ ] shutdown
* [X] sleep
* [ ] sort
* [ ] stat
* [ ] strace
* [ ] su
* [ ] sync
* [ ] tail
* [ ] tee
* [ ] test-bindtodevice
* [ ] test-compress
* [ ] test-crypto
* [ ] test_efault
* [ ] test_env
* [ ] test-fuzz
* [ ] test_io
* [ ] test-pthread
* [ ] test-unveil
* [ ] test-web
* [ ] top
* [ ] touch
* [ ] traceroute
* [ ] tr
* [ ] tree
* [ ] true
* [ ] truncate
* [ ] tt
* [ ] tty
* [X] umount
* [ ] uniq
* [ ] uptime
* [ ] utmpupdate
* [ ] wc
* [ ] w
* [ ] which
* [ ] whoami
* [ ] yes

### CODE REVIEW

**Comment by bcoles** [comment]
I'm not sure how you generated the list of missing man pages, but `pape` appears in both lists.

File: Base/usr/share/man/man1/notify.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Name
+
+notify - create a nofitication
```
Comment (bcoles) [inline Base/usr/share/man/man1/notify.md:3]:
```suggestion
notify - create a notification
```

File: Base/usr/share/man/man1/notify.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Synopsis
+
+```**sh
+$ notify <title> <message> [icon-path]
```
Comment (bcoles) [inline Base/usr/share/man/man1/notify.md:8]: Current convention is to add an `Arguments` heading for positional arguments and an `Options` heading for command line options.

```
$ grep -rn '## Arguments' Base/usr/share/man/
Base/usr/share/man/man1/jp.md:22:## Arguments
Base/usr/share/man/man1/TextEditor.md:20:## Arguments:
Base/usr/share/man/man1/Inspector.md:11:## Arguments
Base/usr/share/man/man1/ls.md:35:## Arguments
Base/usr/share/man/man1/Profiler.md:30:## Arguments
Base/usr/share/man/man1/Playground.md:11:## Arguments
Base/usr/share/man/man1/QuickShow.md:15:## Arguments
Base/usr/share/man/man8/sysctl.md:21:## Arguments
```

```
$ grep -rn '## Options' Base/usr/share/man/
Base/usr/share/man/man1/echo.md:16:## Options
Base/usr/share/man/man1/gml-format.md:15:## Options
Base/usr/share/man/man1/test.md:81:## Options
Base/usr/share/man/man1/date.md:16:## Options
Base/usr/share/man/man1/Shell.md:25:## Options
Base/usr/share/man/man1/md.md:18:## Options
Base/usr/share/man/man1/base64.md:16:## Options
Base/usr/share/man/man1/jp.md:17:## Options
Base/usr/share/man/man1/watch.md:16:## Options
Base/usr/share/man/man1/Terminal.md:15:## Options
Base/usr/share/man/man1/readlink.md:15:## Options
Base/usr/share/man/man1/adjtime.md:24:## Options
Base/usr/share/man/man1/js.md:22:## Options
Base/usr/share/man/man1/uname.md:16:## Options
Base/usr/share/man/man1/xargs.md:30:## Options
Base/usr/share/man/man1/syscall.md:15:## Options
Base/usr/share/man/man1/tar.md:18:## Options
Base/usr/share/man/man1/test-js.md:24:## Options
Base/usr/share/man/man1/TextEditor.md:16:## Options
Base/usr/share/man/man1/expr.md:60:## Options
Base/usr/share/man/man1/find.md:21:## Options
Base/usr/share/man/man1/mktemp.md:18:## Options
Base/usr/share/man/man1/mkdir.md:15:## Options
Base/usr/share/man/man1/crash.md:17:## Options
Base/usr/share/man/man1/ls.md:17:## Options
Base/usr/share/man/man1/pmap.md:15:## Options
Base/usr/share/man/man1/Profiler.md:26:## Options
Base/usr/share/man/man1/pape.md:16:## Options
Base/usr/share/man/man1/avol.md:15:## Options
Base/usr/share/man/man1/EchoServer.md:15:## Options
Base/usr/share/man/man8/userdel.md:17:## Options
Base/usr/share/man/man8/useradd.md:19:## Options
Base/usr/share/man/man8/TelnetServer.md:16:## Options
Base/usr/share/man/man8/sysctl.md:17:## Options
Base/usr/share/man/man8/purge.md:16:## Options
Base/usr/share/man/man5/SystemServer.md:19:## Options
```

File: Base/usr/share/man/man1/sleep.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+## Description
+
+`sleep` will pause execution for specified amount of seconds.
+
+## Examples
+
+```sh
+# sleep 5
```
Comment (bcoles) [inline Base/usr/share/man/man1/sleep.md:18]: `#` usually implies a privileged shell is required. `sleep` does not require `root`.

```suggestion
$ sleep 5
```

**Comment by sppmacd** [comment]
> I'm not sure how you generated the list of missing man pages, but `pape` appears in both lists.

existing: `for f in $(find -type f); do find ../../Base/usr/share/man/ -name $(basename ${f} .cpp).md 2>/dev/null; done | sort > existing-manpages.txt` run from `Userland/Utilities`
then missing: `ls -I $(cat existing-manpages.txt) . | grep .cpp`

File: Base/usr/share/man/man8/umount.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+## Description
+
+`umount` run as root will unmount a mount point specified in argument.
+
+## Examples
+
+```sh
+# umount
```
Comment (bcoles) [inline Base/usr/share/man/man8/umount.md:18]: The `umount` command does nothing without arguments.

File: Base/usr/share/man/man1/notify.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+## Description
+
+`notify` will create a notification with title `title` and message `message`. You can also provide an icon path; by default, no icon will be used.
```
Comment (bcoles) [inline Base/usr/share/man/man1/notify.md:13]:
```suggestion
`notify` will create a WindowServer notification with title `title` and message `message`. You can also provide an icon path; by default, no icon will be used.
```

File: Base/usr/share/man/man1/notify.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+## Examples
+
+```sh
+# <command> && notify "Information" "Command succeeded" /res/icons/32x32/msgbox-information.png
```
Comment (bcoles) [inline Base/usr/share/man/man1/notify.md:18]:
```suggestion
$ <command> && notify "Information" "Command succeeded" /res/icons/32x32/msgbox-information.png
```

**Comment by sppmacd** [comment]
ok, this seems working: `ls -I $(cat existing-manpages.txt | sed 's/^/-I /') . | grep .cpp`
it's now updated

File: Base/usr/share/man/man1/notify.md (hunk 6 above)
Comment (sppmacd) [reply Base/usr/share/man/man1/notify.md]: Fixed.

File: Base/usr/share/man/man1/notify.md (hunk 5 above)
Comment (sppmacd) [reply Base/usr/share/man/man1/notify.md]: Fixed.

File: Base/usr/share/man/man8/umount.md (hunk 4 above)
Comment (sppmacd) [reply Base/usr/share/man/man8/umount.md]: Fixed.

File: Base/usr/share/man/man1/sleep.md (hunk 3 above)
Comment (sppmacd) [reply Base/usr/share/man/man1/sleep.md]: Fixed.

File: Base/usr/share/man/man1/notify.md (hunk 2 above)
Comment (sppmacd) [reply Base/usr/share/man/man1/notify.md:8]: Fixed.

File: Base/usr/share/man/man1/notify.md (hunk 1 above)
Comment (sppmacd) [reply Base/usr/share/man/man1/notify.md]: Fixed.

**Comment by bcoles** [comment]
The `html` utility has been removed in b08d4ccd56e156c72e97218ebe260e83350e4c5a. It can be removed from the list.

**Comment by sppmacd** [comment]
Also, I have a question: Should we really document tests?

**Pull request closed without merging**